src/js/controller/model.js

```diff
@@ -219,6 +219,10 @@ class Model extends SimpleModel {
         if (floating && !!floating.disabled) {
             delete cfg.floating;
         }
+
+        if (typeof cfg.abHideNextUpDisplay !== 'undefined') {
```

Comment:
I opted to add to the `_normalizeConfig` method in the model for the same reason as `floating`. It centralizes this logic and ensures that any downstream consumers of the model config do not have to know or care about ab tests. It also (potentially) centralizes our ab config code to make it easier in future to clean up

Reply:
But config.js doesn't remove or chan `nextUpDisplay, it only ensures a default of true.

I don't see how this helps if we want to avoid allowing a test to override an option specified by the user.

Reply:
The ab-test PR would be a follow-on however your comment makes me think that these changes aren't necessary at all. The ab code can simply set `nextUpDisplay: true/false` depending on the variant selected. Will close